fastestimator/dataset/dataloader.py

```diff
@@ -156,6 +163,13 @@ def _get_fe_iterator(self):
                 return _MPPostBatchIter(self)
             return _MPPreBatchIter(self)
 
+    def __len__(self):
+        return self.fe_samples_to_yield
+
+    @property
+    def batch_size(self):
```

Comment:
this might not work properly since I think the underlying data loader class has it's own batch_size attribute which we don't want to overwrite

Reply:
we'll need to verify that though, not 100% confident